src/SIM/SIM2D.C

```diff
@@ -81,7 +81,7 @@ bool SIM2D::addConnection(int master, int slave, int mEdge, int sEdge, bool reve
                <<" reversed? "<< rever << std::endl;
     ASMs2D* spch = static_cast<ASMs2D*>(myModel[lslave-1]);
     ASMs2D* mpch = static_cast<ASMs2D*>(myModel[lmaster-1]);
-    if (!spch->connectPatch(sEdge,*mpch,mEdge,rever,coordCheck))
+    if (!spch->connectPatch(sEdge,*mpch,mEdge,rever,0,coordCheck))
       return false;
   }
```

Comment:
Foreslår at hele denne commiten, altså  02829b11499e6502bb8ee9a8a4eb1238a6dde4b6 (bortsett fra SIM2D og SIM3D endringen) slås sammen med 8cf83850edd888af5836a724e65a7cc650b876fe, da de toucher stor sett de samme kodelinjene og begge dreier seg om å støtte periodiske betingelser. SIM-endringene kan da naturlig tas i d389ff51b4c2ff959ea1facd45afb1f3829b065e som kommer etterpå.

(her savner jeg mulighet i github for legge til enn generell kommentar som gjelder hele comitten, ikke en bestemt linje, så da ble den slik)